### PR TITLE
Add ipv6calc: calculate/manipulate (not only) IPv6 addresses

### PR DESCRIPTION
**Pull request by nckx**

### CODE REVIEW

File: pkgs/tools/networking/ipv6calc/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    longDescription = ''
+      ipv6calc is a small utility to manipulate (not only) IPv6 addresses and
+      is able to do other tricky things. Intentions were convering a given
+      IPv6 address into compressed format, convering a given IPv6 address into
+      the same format like shown in /proc/net/if_inet6 and (because it was not
+      difficult) migrating the Perl program ip6_int into.
+      Now only one utiltity is needed to do a lot.
+    '';
+    homepage = http://www.deepspace6.net/projects/ipv6calc.html;
+    license = with licenses; gpl2;
```
Comment (iElectric) [inline pkgs/tools/networking/ipv6calc/default.nix]: also set `platforms`

Comment (nckx) [inline pkgs/tools/networking/ipv6calc/default.nix]: Right. I wasn't sure what to do here: it *should* build on all Unixen, even Darwin, but I've no idea if it does.

Restricting it to Linux seems excessive. `all` might be a lie. What is preferred?

Is there a way for me to test this myself in future without bothering someone or breaking something?

Comment (iElectric) [inline pkgs/tools/networking/ipv6calc/default.nix]: `linux` is fine if you can't be assure for the rest.

Comment (nckx) [inline pkgs/tools/networking/ipv6calc/default.nix]: (Done, if that didn't ping you already.)

**Pull request merged**